python/ray/train/base_trainer.py

```diff
@@ -487,5 +490,14 @@ def default_resource_request(cls, config):
                 )
                 return validated_scaling_config.as_placement_group_factory()
 
+        return TrainTrainable
+
+    def as_trainable(self) -> Type["Trainable"]:
+        """Convert self to a ``tune.Trainable`` class."""
+        from ray import tune
```

Comment:
Import at top?

Reply:
All of the existing tune imports are delayed for this class...probably doesn't make too much of a difference but I think it's just to not introduce a Tune dependency unless needed.

Reply:
Ok sg!